sxkeychain erase

```diff
@@ -0,0 +1,22 @@
+credential.helper=osxkeychain
+init.defaultbranch=main
+user.name=eric052199
+user.email=eric052199@gmail.com
+filter.lfs.process=git-lfs filter-process
+filter.lfs.required=true
+filter.lfs.clean=git-lfs clean -- %f
+filter.lfs.smudge=git-lfs smudge -- %f
+core.repositoryformatversion=0
+core.filemode=true
+core.bare=false
+core.logallrefupdates=true
+core.ignorecase=true
+core.precomposeunicode=true
+remote.upstream.url=https://github.com/JabRef/jabref.git
+remote.upstream.fetch=+refs/heads/main:refs/remotes/upstream/main
+branch.main.remote=upstream
+branch.main.merge=refs/heads/main
+remote.origin.url=https://github.com/eric052199/jabref.git
+remote.origin.fetch=+refs/heads/*:refs/remotes/origin/*
+branch.eric-dev.remote=origin
+branch.eric-dev.merge=refs/heads/eric-dev
```

Comment:
This seems to be an artifact of you working on your PR?

Reply:
Sorry, i'm not really sure what this is about. Do i need to remove this?

Reply:
Yes, remove this